rllib/core/rl_module/rl_module.py

```diff
@@ -378,33 +401,126 @@ def get_state(self) -> Mapping[str, Any]:
     def set_state(self, state_dict: Mapping[str, Any]) -> None:
         """Sets the state dict of the module."""
 
-    def serialize(self) -> Mapping[str, Any]:
-        """Return the serialized state of the module."""
-        return {
-            "class": self.__class__,
-            "config": self.config.to_dict(),
-            "state": self.get_state(),
-        }
+    def _save_module_metadata(
```

Comment:
how does this work for nested RLModules?

Reply:
Wait is there such a thing as a nested RLModule? If you mean MARL module, I'm implementing the same methods but for that class as well.